sw/device/silicon_creator/rom_ext/bootstrap.c

```diff
@@ -99,5 +99,48 @@ rom_error_t rom_ext_bootstrap(void) {
   }
   HARDENED_CHECK_EQ(enabled, kHardenedBoolTrue);
 
+  enum {
+    kNumPagesInRomExt = CHIP_ROM_EXT_SIZE_MAX / FLASH_CTRL_PARAM_BYTES_PER_PAGE
```

Comment:
nit: I think we should round this up, right?

Reply:
Do you anticipate that the ROM_EXT size would ever be a non-integer multiple of the page size?

WDYT about just adding a `static_assert`?

```c
 static_assert(CHIP_ROM_EXT_SIZE_MAX % FLASH_CTRL_PARAM_BYTES_PER_PAGE == 0,
               "ROM_EXT size must be an integer multiple of flash page size.");
```

Reply:
No, I don't think we would ever do that. I agree, `static_assert` is better.